keps/sig-network/1880-multiple-service-cidrs/README.md

```diff
@@ -747,22 +729,17 @@ This section must be completed when targeting beta to a release.
 
 ###### How can a rollout or rollback fail? Can it impact already running workloads?
 
-<!--
-Try to be as paranoid as possible - e.g., what if some components will restart
-mid-rollout?
+The objects consumed by the cluster components are the Services, and ClusterIP is a field of these objects.
+Running workloads can not be impacted since the existing Services objects are never mutated.
 
-Be sure to consider highly-available clusters, where, for example,
-feature flags will be enabled on some API servers and not others during the
-rollout. Similarly, consider large clusters and how enablement/disablement
-will rollout across nodes.
--->
+A rollout failure can impact the new apiserver to not be able to start or the clients not being able to create new Services.
 
 ###### What specific metrics should inform a rollback?
 
-<!--
-What signals should users be paying attention to when the feature is young
-that might indicate a serious problem?
--->
```

Comment:
Suggestion:

> There are two key failure modes
> 1) The apiserver unable to start. This can be detected during the rollout phase, no metrics necessary.
> 2) Inability to create services. This can be monitored via ipallocator metrics [fill in the actual metrics]

It took me a bit to parse this section and I don't think the go source paths are appropriate versus mentioning what metrics to look for (and how to use them to identify the failure)